### PR TITLE
Bluetooth: shell: Fix missing return statement for name command

### PR DESCRIPTION
**Pull request by sdalu**
Return once bt name has been displayed

Signed-off-by: Stephane D'Alu <sdalu@sdalu.com>

### CODE REVIEW

**Comment by jhedberg** [comment]
Looks like a good fix, thanks! Just one small thing: could you make the commit message subject consistent with the rest of the Bluetooth commits in the git history, i.e. make it something like "Bluetooth: shell: Fix missing return statement for name command"

File: boards/arm/nrf52840_mdk/board.cmake [hunk 1]
```diff
@@ -2,3 +2,4 @@
 
 board_runner_args(pyocd "--target=nrf52")
 include(${ZEPHYR_BASE}/boards/common/pyocd.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
```
Comment (jhedberg) [inline boards/arm/nrf52840_mdk/board.cmake]: This wasn't part of your previous revision. Was adding it a mistake? It should either way be in a separate commit.

Comment (sdalu) [reply boards/arm/nrf52840_mdk/board.cmake]: oops, commit gone wrong

**Review by jhedberg — APPROVED** [review]
